### PR TITLE
Add current phase and current status to new project mutation#7635

### PR DESCRIPTION
**Pull request by sergiogcx**
## Associated issues

Precludes: https://github.com/cityofaustin/atd-moped/pull/475

Closes https://github.com/cityofaustin/atd-data-tech/issues/7635

## Testing

https://deploy-preview-471--atd-moped-main.netlify.app/moped/projects/new

**Steps to test:**
Create a new project (either a signal project or otherwise), you should see the "potential" phase as the default.

---
#### Ship list
- [x] Code reviewed 
- [x] Product manager approved
- [x] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#gid=776973707)

### CODE REVIEW

**Comment by sergiogcx** [comment]
@amenity @zappfyllc requesting review from you guys to make sure this is the expected behavior, this before I present it to the devs.

**Comment by amenity** [comment]
@sergiogcx - I tested new signals projects and non-signals projects, both got the phase "Potential" added beautifully. As far as status, I'm not sure what I should be seeing since we don't have the badge implemented yet... But maybe @johnclary can confirm this is right?

File: moped-editor/src/queries/project.js [hunk 1]
```diff
@@ -13,6 +13,13 @@ export const ADD_PROJECT = gql`
       fiscal_year
       capitally_funded
       start_date
+      moped_proj_phases {
+        phase_name
+        is_current_phase
+        status_id
+        completion_percentage
+        completed
+      }
```
Comment (johnclary) [inline moped-editor/src/queries/project.js:22]: graphql is incredible

**Review by amenity — APPROVED** [review]

**Comment by johnclary** [comment]
@sergiogcx looks great. can you also populate the start date as the current date when you create the phase? 
Screen Shot 2021-11-23 at 12 46 47 PM


@amenity i haven't kept up with how status badge will be implemented, but Sergio is setting the project status to `1` ("active", i believe) which looks correct to me and can be easily changed.

**Review by johnclary — APPROVED** [review]
🚢

**Review by frankhereford — APPROVED** [review]
Tested two ways: by creating a signal based project and a non-signal based project - both worked great. I also went through the code, and I think it's right on point.  🚢 👍

**Review by mateoclarke — APPROVED** [review]
🚢 💨 ✨

**Pull request merged**